solutions/usaco-988.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-988
+source: USACO Bronze 2020 January
+title: Photoshoot
+author: Ananth Kashyap
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
```

Comment:
if I'm not wrong your implementation is O(N^3)
> if ((b[j] - a[j]) in a)

^^ this takes O(N) and you have two nested loops

Reply:
so please change it to O(N^2)

Reply:
```suggestion
**Time Complexity:** $\mathcal{O}(N^3)$
```

Reply:
@jessechoe10 why?